src/test/java/com/google/cloud/spanner/jdbc/SpannerJdbcExceptionMatcher.java

```diff
@@ -26,19 +26,12 @@
   private final Code errorCode;
   private final String message;
 
-  public static <T extends JdbcSqlException> SpannerJdbcExceptionMatcher<T> matchCode(
```

Comment:
Is this a breaking change?

Reply:
Yes, I guess it is if someone also imports the test library and extends this. I've re-added this method.